Source/Plugin.BLE.iOS/Device.cs

```diff
@@ -135,5 +135,11 @@ public void Update(CBPeripheral nativeDevice)
             Trace.Message($"Request MTU is not supported on iOS.");
             return await Task.FromResult((int)_nativeDevice.GetMaximumWriteValueLength(CBCharacteristicWriteType.WithoutResponse));
         }
+
+        protected override async Task<bool> UpdateConnectionIntervalNativeAsync(ConnectionInterval interval)
+        {
+            Trace.Message("Cannot update connection inteval on iOS. Returning TRUE ..");
+            return await Task.FromResult(true);
```

Comment:
return true; when async
or
return Task.FromResult(true); when not async. I'd prefer this one, because it doesn't generate a state machine.

Question: Why not false? False would be better, or? If you compare it to the Android implementation.